sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## 1.2.0 (Unreleased)
```

Comment:
Why did we jump from 1.1.0b1 to 1.2.0? Shouldn't the next GA release be 1.1.0?

Reply:
yes, it should be 1.1.0, I will fix it. Thanks!

Reply:
This release will include dropping Python 2.x support which is a minor change([#20691](https://github.com/Azure/azure-sdk-for-python/issues/20691)), so that is why we jumped to 1.2.0

Reply:
1.1.0 Never GA'ed - so you can roll the dropping of 2.7 into that release and it will still be part of a minor bump :)